giskard/slicing/tree_slicer.py

```diff
@@ -79,24 +80,34 @@ def find_slices(self, features, target=None):
                     "min_impurity_decrease": np.linspace(data_var / 100, data_var / 10, 10)
                 },
             )
+            gs.fit(data.loc[:, features], data.loc[:, target])
+            dt = gs.best_estimator_
         else:
             logger.debug("Target is not numeric, using a classification tree.")
             dt = DecisionTreeClassifier(
                 criterion="gini",
                 splitter="best",
                 min_samples_leaf=min_samples,
                 max_leaf_nodes=20,
+                min_impurity_decrease=0,
             )
-            gs = GridSearchCV(dt, {"min_impurity_decrease": np.linspace(0.001, 0.1, 10)})
-
-        gs.fit(data.loc[:, features], data.loc[:, target])
-        dt = gs.best_estimator_
+            dt.fit(data.loc[:, features], data.loc[:, target])
 
         # Need at least a split, otherwise return now.
         if dt.tree_.node_count < 2:
             logger.debug("No split found, stopping now.")
             return []
 
+        analytics.track(
+            "scan:dt_params",
```

Comment:
let's make this more explicit or we won't know what this is

Reply:
Maybe we should remove it, we don't need it anymore